BookDaoInterface.java

```diff
@@ -0,0 +1,10 @@
+import java.util.Map;
+
+public interface BookDaoInterface {
```

Comment:
I think you dont need to use book in names of method since it is clear from Class Name